src/pages/signin.tsx

```diff
@@ -34,12 +39,31 @@ export default function Signin() {
   });
 
   const [serverErrorState, setServerError] = useState<string | null>(null);
+  const dispatch = useAppDispatch();
+  const currentUserid = useAppSelector(selectUserid);
+  const router = useRouter();
 
-  const onSubmitHandler: SubmitHandler<FormValues> = async () => {
+  const onSubmitHandler: SubmitHandler<FormValues> = async (formData) => {
     try {
       // @TODO Implement submit
       // await onSubmit(formData.email, formData.password);
 
+      const signInPayload = { email: formData.email, password: formData.password };
+      const signInRes = await axiosSignIn('/api/v1/auth/signin', signInPayload);
```

Comment:
Generalize the function a little bit more, if you are making a separete function just to sing in, doesn't need a dynamic path.

```suggestion
      const { data } = await signInWithEmailAndPassword({ email, password });
```